### ISSUE TITLE
I would like to suggest more sounds when the time expires.

### ISSUE THREAD

**Issue by Pantini** [Splode/pomotroid#268]
Like a school bell ringing
Or anything really, cause the default sounds are so subtle that sometimes I miss them. C:

**Comment by albertoeda** [comment]
I agree! A selector for additional sounds would be great

Also, the ability to choose a custom sound would be even better! Something simple in the GUI for us less-technical users, or maybe a folder where you can drop sound files in the appropriate format

**Comment by maido-39** [comment]
Agree with this opinion!!

**Comment by jupiter104** [comment]
Yes, we really need this. I want to play my own music files during the break or during the focus.

**Comment by Splode** [comment]
Custom audio for round completion is now available in [v1.0.0](https://github.com/Splode/pomotroid/releases/tag/v1.0.0).
